lldb/docs/use/python.rst

```diff
@@ -11,6 +11,65 @@ some of these things by going through an example, explaining how to use
 Python scripting to find a bug in a program that searches for text in a
 large binary tree.
 
+Operating System Thread Plugins
+-------------------------------
+
+.. automodapi:: lldb.plugins.operating_system
+    :no-heading:
+    :members:
+    :undoc-members:
+    :skip: command
+    :skip: declaration
+    :skip: in_range
+    :skip: is_numeric_type
+    :skip: lldb_iter
+    :no-inheritance-diagram:
+
+Scripted Process Plugins
+-------------------------------
+
+.. automodapi:: lldb.plugins.scripted_process
+    :no-heading:
+    :members:
+    :undoc-members:
+    :skip: ABCMeta
+    :skip: command
+    :skip: declaration
+    :skip: in_range
+    :skip: is_numeric_type
+    :skip: lldb_iter
+    :no-inheritance-diagram:
```

Comment:
I don't think you meant to add this to the [Python Scripting](https://lldb.llvm.org/use/python.html) page as this is a tutorial. The automatically generated documentation should probably end up on the [Python API](https://lldb.llvm.org/python_api.html) page. But you probably also want to have a page that explains what these things are, and how you'd use them. It seems like the [Python Reference](https://lldb.llvm.org/use/python-reference.html) page would be best suited for that.

Reply:
We really should refresh the Python Scripting page, having more task-focused docs would be good.  But that's pretty old at this point, and focused on a single problem, so I agree with Jonas, it's not the right place to put this info.